docs/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md

```diff
@@ -64,16 +64,16 @@ Policy
 
 ## Additional resources
 
-- **Retry pattern**  
+- **Retry pattern** \
```

Comment:
we've had problems with loc using this formatting before. I'd keep as-is

Reply:
Ok, What's better (or "less bad") using `<br />` or two (invisible) spaces?

Reply:
I realized you're using two spaces in other PR so there goes another commit 😊